openlibrary/core/edits.py

```diff
@@ -143,8 +143,19 @@ def submit_work_merge_request(
                 comment=comment,
                 reviewer=reviewer,
                 status=status,
+                title=cls.get_work_merge_title(work_ids),
             )
 
+    @classmethod
+    def get_work_merge_title(cls, olids):
```

Comment:
The class isn't used inside this scope so we can use `staticmethod` instead of `classmethod` 

```suggestion
    @staticmethod
    def get_work_merge_title(olids):
```

Reply:
Whoops!  Not sure how I missed that... Thanks!